plugins/caretposition/plugin.js

```diff
@@ -0,0 +1,168 @@
+/**
+ * @license Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+'use strict';
+
+( function() {
+
+	CKEDITOR.plugins.add( 'caretposition', {} );
```

Comment:
Actually now that we have `range.getClientRects()` do we need this plugin at all?